features/steps/markdown2html_steps.py

```diff
@@ -15,11 +13,8 @@
 def normalize_html(html_text):
     html_text = '\n'.join(html.tostring(el, encoding=str)
                            for el in antimarkdown.parse_fragments(html_text))
-    tidied = tidy(
-        '\n'.join(line.strip()
-                   for line in nodes.whitespace(html_text.strip()).replace('>', '>\n').splitlines()),
-        pretty_print=True)
-    return tidied.decode('utf-8')
+    return '\n'.join(line.strip()
+                     for line in nodes.whitespace(html_text.strip()).replace('>', '>\n').splitlines())
```

Comment:
You're probably going to want some sort of `tidy()` function so you get sane diffs for any broken tests. BeautifulSoup has decent tree normalization, or you could use Derek's [tidy-alike](https://github.com/Crossway/antimarkdown/pull/6/files#diff-463b9c97c640c5e992540b7ca49b3902R17), though I haven't evaluated its effectiveness yet.